### PR TITLE
Update DD.html

### PR DESCRIPTION
**Pull request by sls-lnluksa**

### CODE REVIEW

File: DD.html [hunk 1]
```diff
@@ -35,6 +35,7 @@
 <body>
 
 <!-- html comment -->
+<!-- html comment -->
```
Comment (sls-lnluksa) [inline DD.html:38]: Yes this is OK

File: DD.html [hunk 2]
```diff
@@ -35,6 +35,7 @@
...
 <p>Drag the W3Schools image into the rectangle:</p>
 
 # table>(tr>td*3>div[ondrop="drop(event)" ondragover="allowDrop(event)"]#div$)+(tr>td*3>div[ondrop="drop(event)"ondragover="allowDrop(event)"]#div$@4)
```
Comment (sls-lnluksa) [inline DD.html:41]: Consider this changed

File: DD.html [hunk 3]
```diff
@@ -35,6 +35,7 @@
 <body>
```
Comment (sls-lnluksa) [inline DD.html:35]: Body should have class

**Review by sls-lnluksa — COMMENTED** [review]
This is poor coding

**Review by stoivo — APPROVED** [review]
Looks awesome

**Review by agvald — CHANGES_REQUESTED** [review]
bad

**Review by agvald — APPROVED** [review]

**Pull request merged**